data/human/human missions.txt

```diff
@@ -4925,3 +4925,147 @@ mission "Riley"
+			`	"Bah, you lying buffoon! I'll find him in your ship's wreckage!"`
+				goto power
+			label negotiate
+			`	"Aye, then I won't negotiate with you!"`
+				goto power
+			label power
+			`	The Vanguard powers up its weapons.`
+	npc kill
+		government "Independent (Killable)"
+		personality heroic vindictive staying
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"		
+		conversation 
+			`The Golden Jewel has been destroyed. You receive a hail from a small escape pod. "Thank you Captain, you've saved my life. I'm James, meet me on <destination> and I'll give you your reward"`
+			choice
+				`	"Sounds good"`
+					accept
+				`	"What's the deal with you and that pirate?"`
+					goto piratedeal
+			label piratedeal
+			`"That pirate was a potential investor with the firm I work for. We had a disagreement over terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to go to <destination>.`
+	on complete
+		"reputation: Independent (Killable)" = 10
+		conversation
+			`When you land on <destination>, James is waiting for you at the Spaceport. "Thank you, truly. I'm James Vendeur, Investor Relations Associate with Maralex Controls. Anyway, I've managed to scrounge up my savings. It's not much to someone like you, I'm afraid. Just over ninety thousand credits."`
+			`	"However, I see you're clearly a captain of means. I'd have to pull a few strings, but what if instead I get you into a private placement of Maralex stock? To purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents. Our historical returns are 20% per annum which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits."`
+			choice
+				`	"Sure, I'll invest. What do I need to do?"`
+					goto invest
+				`	"Maralex Controls? I thought you were part of the Syndicate."`
+					goto maralex
+				`	"Private placement? Does that mean I own part of the Syndicate?"`
+					goto placement
+				`	"No thanks, I'll take the money please."`
+					goto payup
+			label maralex
+			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We develop innovate micro-control solutions for high-performance commercial and military applications for vehicles traveling by sea, air, and of course space. Succinctly, we primarily develop fighter engines and drag designs. Would you like to invest?"`
+			choice
+				`	"Sure, I'll invest. What do I need to do?"`
+					goto invest
+				`	"Nope, I'll stick with the 90k."`	
+					goto payup
+			label placement
+			`	"Almost, you will become a part owner of a partially owned subsidiary of Syndicated Systems -- Maralex Controls. As a privately held business, there are few opportunities to invest, and even fewer people who are permitted to. It's quite an exclusive investment, with quite high returns, I might add. Would you like to invest with us?"`
```

Comment:
```suggestion
			`	"Almost; you will become a part owner of a partially owned subsidiary of Syndicated Systems -- Maralex Controls. As a privately held business, there are few opportunities to invest, and even fewer people who are permitted to. It's quite an exclusive investment, with quite high returns, I might add. Would you like to invest in us?"`
```
Not sure if `--` should stay; probably a single one is enough here.

Reply:
Fixed/moot and I agree on the --